rest-service/manager_rest/configure_manager.py

```diff
@@ -25,6 +31,75 @@ def _generate_password(length=12):
     return password
 
 
+def _insert_rabbitmq_broker(brokers, ca_id):
+    sm = get_storage_manager()
+
+    for broker in brokers:
+        inst = models.RabbitMQBroker(
+            _ca_cert_id=ca_id,
+            **broker
+        )
+        sm.put(inst)
+
+
+def _create_rabbitmq_info(rabbitmq_config):
+    use_hostnames = rabbitmq_config['use_hostnames_in_db']
+    is_external = rabbitmq_config.get('is_external', False)
+
+    return [
+        {
+            'name': name,
+            'host': name if use_hostnames else broker['networks']['default'],
+            'management_host': (
+                name if use_hostnames else broker['networks']['default']
+            ),
+            'username': rabbitmq_config['username'],
+            'password': rabbitmq_config['password'],
+            'params': None,
+            'networks': broker['networks'],
+            'is_external': is_external,
+        }
+        for name, broker in rabbitmq_config['cluster_members'].items()
+    ]
+
+
+def _get_rabbitmq_ca_cert(rabbitmq_ca_cert_path):
+    if rabbitmq_ca_cert_path:
+        with open(rabbitmq_ca_cert_path) as f:
+            return f.read()
+
+    return ''
+
+
+def _insert_rabbitmq_ca_cert(cert, name):
+    sm = get_storage_manager()
+    inst = models.Certificate(
+        name=name,
+        value=cert,
+        updated_at=datetime.datetime.now(),
+        _updater_id=0,
+    )
+    sm.put(inst)
+
+    return inst.id
```

Comment:
hmm, if we returned `inst` directly rather than `inst.id`, then creating `models.RabbitMQBroker(` could take `ca_cert=this_inst` rather than `_ca_cert_id`

The difference is that `.id` is not necessarily assigned immediately, only on commit, so we could have this not require a commit
(currently `sm.put()` commits)